### DISCUSSION TITLE
Add Waveshare RS485 TO POE ETH (B) to adapter list

### DISCUSSION THREAD

**Discussion by silkit** [nathanmarlor/foxess_modbus#642] [Ideas]
Could we please add the waveshare RS485 TO POE ETH (B) module to the initial set up screen:
![image](https://github.com/nathanmarlor/foxess_modbus/assets/33449032/d4e06ddc-a6b1-4f56-a576-00458c4d8f63)

Thanks

**Comment by horstvogel** [comment]
Is the not the same like the https://github.com/nathanmarlor/foxess_modbus/wiki/Waveshare-RS485-to-ETH-%28B%29  ?

**Reply by silkit** [reply]
Yes that is the same module.
With my experience as a new user i was hoping it could be included as an adapter type under the initial set-up screen so it would "use settings tailored to your adapter" rather than manually configuring them and it says if you don't see your model then select "other" and let us know so i was just suggesting that it was included in that list and there was also no link from the set-up wizard.
The other listed adapters on github wiki are in the list above, but not this one.

**Reply by DerRonnyUK** [reply]
I agree with the above - make it simple for new starter to this. the naming is a bit confusing :)
